src/views/v2/rsvp-kitchen-sink/default-no-description.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Block: RSVP default no description
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp-kitchen-sink/default-no-description.php
+ *
+ * See more documentation about our Blocks Editor templating system.
+ *
+ * @link {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @var Tribe__Tickets__Ticket_Object $rsvp The rsvp ticket object.
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+?>
```

Comment:
:no_entry_sign: There must be exactly one blank line after the file comment

<sub>hash: 573a1bd3748f8c1a336fec2ddb6f5ec7</sub>